pages/guide.py

```diff
@@ -623,7 +623,7 @@ def update_stage_fields(self, feature_id: int, feature_type: int,
         stage.milestones = milestoneset_entity
       # If the field starts with "intent_", it should modify the
       # more general "intent_thread_url" field.
-      elif field.startswith('intent_'):
+      elif field.startswith(('intent_', 'ready_for_trial_url')):
```

Comment:
We'll need to change this in the migration script as well. Currently, this field is saved on Dev Trial stage entities as `announcement_url`. I'm not sure if this was a mistake on my part, but I'll create a PR changing this.

Reply:
Oh, announcement_url sounds like the better place to put it.  I forgot what we had previously said about that field.  I'll change this PR.

Reply:
Please check that I have it right this time.